tests/testthat/test-get_map_estimates.ss.R

```diff
@@ -84,13 +82,25 @@ test_that("MAP works with linear steady state equations", {
   interval <- 24
   reg <- PKPDsim::new_regimen(amt = 1000, n = 30, interval = interval, type = "oral")
   t_tdm <- max(reg$dose_times) + c(1, 3, 6, 8, 12)
+  covariates <- list(
+    WT = PKPDsim::new_covariate(
+      value = c(70, 120), 
+      times = max(reg$dose_times) + c(0, 120)
+    )
+  )
+  covariates_ss <- list(
+    WT = PKPDsim::new_covariate(
+      value = c(70, 120), 
+      times = c(0, 120)
+    )
+  )
```

Comment:
We should create `model2` here instead of line 68 since it relies on the `covariates` object:

```suggestion
  covariates <- list(
    WT = PKPDsim::new_covariate(
      value = c(70, 120), 
      times = max(reg$dose_times) + c(0, 120)
    )
  )
  covariates_ss <- list(
    WT = PKPDsim::new_covariate(
      value = c(70, 120), 
      times = c(0, 120)
    )
  )
  model2 <- PKPDsim::new_ode_model(
    code = "
      CLi = CL * pow(WT/70.0, 0.75)
      dAdt[1] = -KA*A[1]
      dAdt[2] = KA*A[1] - (CLi/V)*A[2]
    ", 
    obs = list(cmt = 2, scale = "V"),
    declare_variables = "CLi",
    dose = list(cmt = 1), 
    covariates = covariates
  )
```